### ISSUE TITLE
[V3] <Column> syntax doesn't seem to work

### ISSUE THREAD

**Issue by hakunin** [zippyui/react-datagrid#221]
According to the Readme, this should work:

        <DataGrid>
          <Column name="shop_item_id" title="ID" width={90} />
          <Column name="price" title="Cena" textAlign="right" width={90} render={money} type="number"/>
        </DataGrid>

But instead I get:

![screenshot 2016-06-17 16 39 48](https://cloud.githubusercontent.com/assets/65846/16154211/22f3d96c-34aa-11e6-8ff6-a73493c0b9c9.png)

(the columns and children are both undefined)

**Comment by radubrehar** [comment]
Will investigate this and come back with a solution on Monday

**Comment by hakunin** [comment]
Thanks, I just tried the syntax because the regular way wasn't working https://github.com/zippyui/react-datagrid/issues/220

**Comment by radubrehar** [comment]
This is fixed now, landed in `3.0.19-beta`
You can nest `Column` or `ColumnGroup` inside a `DataGrid`

```jsx
import DataGrid, { Column } from 'react-datagrid'
<DataGrid idProperty="id">
   <Column name="firstName" />
   <Column name="lastNameName" />
</DataGrid>
```
or
```jsx
import DataGrid, { Column, ColumnGroup } from 'react-datagrid'
<DataGrid idProperty="id">
  <ColumnGroup fixed>
     <Column name="firstName" />
     <Column name="lastNameName" />
  </ColumnGroup>
  <ColumnGroup >
     <Column name="age" />
     <Column name="gender" />
  </ColumnGroup>
</DataGrid>
```
